.github/workflows/awstest.yml

```diff
@@ -1,11 +1,10 @@
 name: nf-core AWS test
 # This workflow is triggered on push to the master branch.
-# It runs the -profile 'test' on AWS batch
+# It can be additionally triggered manually with GitHub actions workflow dispatch.
+# It runs the -profile 'test' on AWS batch.
 
 on:
-  push:
-    branches:
-      - master
+  workflow_dispatch:
```

Comment:
no master?

Reply:
https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/.github/workflows/awsfulltest.yml#L6

Not in the template.
The `workflow_dispatch` allows you to trigger it manually. 
Hence, you can trigger it once before releasing and after the release has been published it gets triggered anyways.